swagger.yaml

```
@@ -3736,6 +3736,11 @@ definitions:
         description: List of OLM operators to be installed.
         items:
           $ref: '#/definitions/operator-create-params'
+      hypethreading_config:
```

```suggestion
      hyperthreading:
```
(Note the missing r in hyperthreading)

---

fixed